pkg/models/commits.go

```diff
@@ -20,6 +20,15 @@ type Commit struct {
 	Packages             []Package `json:"Packages" gorm:"many2many:commit_packages;"`
 	ComposeJobID         string    `json:"ComposeJobID"`
 	Status               string    `json:"Status"`
+	RepoID               uint      `json:"RepoID"`
```

Comment:
```suggestion
	RepoID               *uint      `json:"RepoID"`
```

This one needs to be a pointer, running against postgresql it violates a fk contrainst.